src/com/googlecode/jmxtrans/model/Query.java

```diff
@@ -78,6 +81,14 @@ public String getResultAlias() {
 	}
 
 	/**
+	 * The useObjDomainAsKey property allows you to specify the use of the Domain portion of the Object Name
+	 * as part of the output key instead of using the ClassName of the MBean which is the default behavior.
+	 */
+	public boolean isUseObjDomainAsKey() {
```

Comment:
`isUse` does not sound like valid english to me (but English is not my first language). Wouldn't `isUsingObjectDomainAsKey` be more correct?